packages/button/src/connectors/rainbow.ts

```diff
@@ -0,0 +1,20 @@
+/*eslint-disable no-redeclare */
+import { connectorsForWallets } from '@rainbow-me/rainbowkit';
+import type { RainbowWalletOptions } from '@rainbow-me/rainbowkit/dist/wallets/walletConnectors/rainbowWallet/rainbowWallet';
+import { rainbowWallet } from '@rainbow-me/rainbowkit/wallets';
+import { Connector } from '@wagmi/core';
+
+declare class RainbowConnector extends Connector {
+  constructor({ chains }: RainbowWalletOptions);
+}
+
+function RainbowConnector({ chains, projectId }: RainbowWalletOptions) {
+  return connectorsForWallets([
+    {
+      groupName: 'Popular',
+      wallets: [rainbowWallet({ chains, projectId })],
+    },
+  ])()[0];
+}
```

Comment:
Yeah that's a good point. Essentially even if we use `getDefaultWallets` it will kind of still rely on `rainbowWallet` rainbow connector and we still have to play around with indexes like `[0]` or `.filter()`. 

What about in the next release creating a function that prepares only one connector for us so it's easier and maintainable ?